Makefile

```diff
@@ -5,7 +5,7 @@ BIN_NAME=terraform-provider-instaclustr
 # for VERSION, don't add prefix "v", e.g., use "1.9.8" instead of "v1.9.8" as it could break circleCI stuff
 
 
-VERSION=1.17.0
+VERSION=1.17.1
```

Comment:
Since this is a new feature, should be a minor version bump not patch

Reply:
yes, fixed